plugins/islandora_context_reaction_restrict_by_ip.inc

```diff
@@ -111,7 +111,7 @@ class IslandoraContextReactionRestrictByIp extends context_reaction {
       if (!strlen($range)) {
         continue;
       }
-      list($low, $high) = explode(':', $range);
+      list($low, $high) = array_pad(explode(':', $range, 2), 2, NULL);
```

Comment:
OK, I'll merge. Do you want to open a new ticket for the discussion about how to represent single IP addresses?